File: public/phase-2-20160620022203/C_SHARP/pony-sequence.cs
// Name : Pony Sequence
using System;
namespace Application {
	class pony_sequence {
		public static void Main(string[] args) {
		// input:
		// n : number of elements in the sequences in a and b
		// a : sequence a
		int index;
		int n = Convert.ToInt32(Console.ReadLine());
		int[] a = new int[n];
		string[] a_elements = (Console.ReadLine()).Split(' ');
		for(index=0;index<n;index++)
			a[index] = Convert.ToInt32(a_elements[index]);


		// write your code here
		// store your results in `b`

		// output
		// Dummy Data
		int[] b = new int[] {1,1,1,1,1};

		Console.WriteLine(string.Join(" ", b));
		}	
	}
}

File: public/phase-2-20160620022203/C_PLUS_PLUS/pony-sequence.cpp
// Name : Pony Sequence
#include <iostream>
#include <string>
using namespace std;

int main(int argc, char const *argv[]) {
	// input:
	// n : number of elements in the sequences in a and b
	// a : sequence a
	int index;
	int n;
	cin >> n;
	int a[100];

	for(index=0;index<n;index++)
		cin >> a[index];


	// write your code here
	// store your results in `b`

	// output
	// Dummy Data
	int b[100]={1,1,1,1,1};

	for(index=0;index<n;index++)
		cout << b[index] << " ";

}